liquibase-standard/src/main/java/liquibase/changelog/AbstractChangeLogHistoryService.java

```diff
@@ -132,7 +132,8 @@ public void resetDeploymentId() {
     public void generateDeploymentId() {
         if (this.deploymentId == null) {
             String dateString = String.valueOf(new Date().getTime());
-            this.deploymentId = dateString.substring(dateString.length() - 10);
+            this.deploymentId = dateString.length() > 9 ? dateString.substring(dateString.length() - 10) :
+                    String.format("%010d", dateString);
```

Comment:
This new piece is not working for me, when setting my local time to `1/1/1970 1:00:00` when this code gets executed I'm getting a `d != java.lang.String` error, because the format is not totally accurate as `d` flag expects for a number, but we are passing over a string. 

I tried something like this:

```
if (this.deploymentId == null) {
            long dateTime = new Date().getTime();
            String dateString = String.valueOf(dateTime);
            DecimalFormat decimalFormat = new DecimalFormat("0000000000");

            this.deploymentId = dateString.length() > 9 ? dateString.substring(dateString.length() - 10) :
                    decimalFormat.format(dateTime);
        }
```
and it worked for me. 

Would you mind updating this code, or if you prefer I can do it?

Thanks,
Daniel.